amy/extforms/views.py

```diff
@@ -79,7 +79,7 @@ class WorkshopRequestCreate(
     page_title = "Request a Carpentries Workshop"
     template_name = "forms/workshoprequest.html"
     success_url = reverse_lazy("workshop_request_confirm")
-    email_fail_silently = False
```

Comment:
Is there a way to do this only in non-production environments?

Reply:
We can make condition based on the environment, just like in the case of top banner in test-amy or local env, but I'm not sure this is a good idea.

If the fail is silenced in test-amy then the forms should work (Maneesha's issue is fixed). In production we never had any issues with Mailgun, so this was never a problem.